extensions/amp-analytics/0.1/events.js

```diff
@@ -138,13 +138,22 @@ export class CustomEventTracker extends EventTracker {
         });
       }, 1);
     }
-
+    let selector = config['selector'];
+    if (!selector) {
+      selector = ':root';
+    }
+    const selectionMethod = config['selectionMethod'] || null;
+    const target = this.root.getElement(context, selector, selectionMethod);
```

Comment:
So, one nuance, but somewhat big: if we need to execute selector - we should wait for document-ready signal. See https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/0.1/events.js#L256 for an example. Otherwise you may try to search for an element to early and find nothing.